### PR TITLE
Update NFT case to camelcase

### PR DESCRIPTION
**Pull request by Steveantor**
Updating from [refactor/general-cleanup](https://github.com/ubiquity/ubiquity-dollar/tree/refactor/general-cleanup)
Resolves #

### CODE REVIEW

File: packages/contracts/test/helpers/LocalTestHelper.sol [hunk 1]
```diff
@@ -33,24 +33,24 @@ abstract contract LocalTestHelper is Test {
...
     DollarMintExcess dollarMintExcess;
 
     function setUp() public virtual {
         manager = new UbiquityDollarManager(admin);
 
         vm.startPrank(admin);
-        // deploy credit NFT token
-        creditNFT = new MockCreditNFT(100);
-        manager.setCreditNFTAddress(address(creditNFT));
+        // deploy credit Nft token
```
Comment (0x4007) [inline packages/contracts/test/helpers/LocalTestHelper.sol]:
```suggestion
        // deploy Credit NFT token
```

File: packages/contracts/test/dollar/core/CreditNftRedemptionCalculator.t.sol [hunk 2]
```diff
@@ -0,0 +1,49 @@
+// SPDX-License-Identifier: MIT
```
Comment (0x4007) [inline packages/contracts/test/dollar/core/CreditNftRedemptionCalculator.t.sol:1]: Where did this file come from?

File: packages/contracts/src/dollar/interfaces/ICreditNftRedemptionCalculator.sol [hunk 3]
```diff
@@ -3,9 +3,9 @@ pragma solidity 0.8.16;
 
 import "@openzeppelin/contracts/token/ERC1155/IERC1155Receiver.sol";
 
-/// @title A mechanism for calculating Credit NFTs received for a dollar amount burnt
-interface ICreditNFTRedemptionCalculator {
-    function getCreditNFTAmount(
+/// @title A mechanism for calculating Credit Nfts received for a dollar amount burnt
```
Comment (0x4007) [inline packages/contracts/src/dollar/interfaces/ICreditNftRedemptionCalculator.sol]:
```suggestion
/// @title A mechanism for calculating Credit NFTs received for a dollar amount burnt
```

File: packages/contracts/src/dollar/interfaces/ICreditNftManager.sol [hunk 4]
```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: MIT
```
Comment (0x4007) [inline packages/contracts/src/dollar/interfaces/ICreditNftManager.sol]: Where did this new file come from?

File: packages/contracts/src/dollar/core/CreditNft.sol [hunk 5]
```diff
@@ -70,22 +70,22 @@ contract CreditNFT is ERC1155Ubiquity, ICreditNFT {
...
-    /// @notice Burn an amount of CreditNFTs expiring at a certain block from
+    /// @notice Burn an amount of CreditNfts expiring at a certain block from
     /// a certain holder's balance
-    /// @param creditNFTOwner the owner of those CreditNFTs
+    /// @param creditNftOwner the owner of those CreditNfts
     /// @param amount amount of tokens to burn
-    /// @param expiryBlockNumber the expiration block number of the CreditNFTs to burn
-    function burnCreditNFT(
-        address creditNFTOwner,
+    /// @param expiryBlockNumber the expiration block number of the CreditNfts to burn
```
Comment (0x4007) [inline packages/contracts/src/dollar/core/CreditNft.sol]:
```suggestion
    /// @param expiryBlockNumber the expiration block number of the CreditNFTs to burn
```

File: packages/contracts/src/dollar/core/CreditNft.sol [hunk 6]
```diff
@@ -70,22 +70,22 @@ contract CreditNFT is ERC1155Ubiquity, ICreditNFT {
         _totalOutstandingDebt = _totalOutstandingDebt + (amount);
     }
 
-    /// @notice Burn an amount of CreditNFTs expiring at a certain block from
+    /// @notice Burn an amount of CreditNfts expiring at a certain block from
     /// a certain holder's balance
-    /// @param creditNFTOwner the owner of those CreditNFTs
+    /// @param creditNftOwner the owner of those CreditNfts
```
Comment (0x4007) [inline packages/contracts/src/dollar/core/CreditNft.sol]:
```suggestion
    /// @param creditNftOwner the owner of those CreditNFTs
```

File: packages/contracts/src/dollar/core/CreditNft.sol [hunk 7]
```diff
@@ -48,16 +48,16 @@ contract CreditNFT is ERC1155Ubiquity, ICreditNFT {
         _totalOutstandingDebt = 0;
     }
 
-    /// @notice Mint an amount of CreditNFTs expiring at a certain block for a certain recipient
+    /// @notice Mint an amount of CreditNfts expiring at a certain block for a certain recipient
     /// @param amount amount of tokens to mint
-    /// @param expiryBlockNumber the expiration block number of the CreditNFTs to mint
-    function mintCreditNFT(
+    /// @param expiryBlockNumber the expiration block number of the CreditNfts to mint
```
Comment (0x4007) [inline packages/contracts/src/dollar/core/CreditNft.sol]:
```suggestion
    /// @param expiryBlockNumber the expiration block number of the CreditNFTs to mint
```

**Review by 0x4007 — CHANGES_REQUESTED** [review]
I think lets get this merged in as soon as possible before merge conflicts inevitably arise.

**Comment by 0x4007** [comment]
Make sure to rebase onto `development` now that we have CI stable (minus Slither lol)

File: packages/contracts/src/dollar/interfaces/ICreditNftManager.sol [hunk 8]
```diff
@@ -0,0 +1,13 @@
...
+pragma solidity 0.8.16;
+
+import "@openzeppelin/contracts/token/ERC1155/IERC1155Receiver.sol";
+
+/// @title A Credit Nft redemption mechanism for Credit Nft holders
```
Comment (0x4007) [inline packages/contracts/src/dollar/interfaces/ICreditNftManager.sol]:
```suggestion
/// @title A Credit NFT redemption mechanism for Credit NFT holders
```

File: packages/contracts/src/dollar/interfaces/ICreditNftManager.sol [hunk 9]
```diff
@@ -0,0 +1,13 @@
...
+/// @notice Allows users to redeem individual Credit Nft or batch redeem Credit Nft
```
Comment (0x4007) [inline packages/contracts/src/dollar/interfaces/ICreditNftManager.sol]:
```suggestion
/// @notice Allows users to redeem individual Credit NFT or batch redeem Credit NFT
```

File: packages/contracts/test/dollar/core/CreditNftRedemptionCalculator.t.sol (hunk 2 above)
Comment (Steveantor) [reply packages/contracts/test/dollar/core/CreditNftRedemptionCalculator.t.sol:1]: They're not new so what I did was
Delete those files on commit 1,
Restore and rename files on commit 2
Thus avoiding the `gitignorecase ` issue.

File: packages/contracts/src/dollar/interfaces/ICreditNftManager.sol (hunk 4 above)
Comment (Steveantor) [reply packages/contracts/src/dollar/interfaces/ICreditNftManager.sol]: They're not new so what I did was
Delete those files on commit 1,
Restore and rename files on commit 2
Thus avoiding the `gitignorecase` issue.

**Pull request merged**